lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1028,7 +1028,7 @@ def test_create_basket(self):
         ecommerce api, we correctly call to that api to create a basket.
         """
         user = UserFactory.create(username="test-username")
-        course_mode = CourseModeFactory(sku="test-sku")
+        course_mode = CourseModeFactory(sku="test-sku").to_tuple()  # pylint: disable=no-member
```

Comment:
`checkout_with_ecommerce_service()` is [fed the output](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/verify_student/views.py#L686) of [`CourseMode.paid_modes_for_course()`](https://github.com/edx/edx-platform/blob/master/common/djangoapps/course_modes/models.py#L166). `paid_modes_for_course()` and other mode retrieval utilities like it don't actually return modes. Instead, they return "mode tuples."